scrapy_zyte_smartproxy/middleware.py

```diff
@@ -10,16 +10,16 @@
 from scrapy.exceptions import ScrapyDeprecationWarning
 from twisted.internet.error import ConnectionRefusedError, ConnectionDone
 
-from scrapy_crawlera.utils import exp_backoff
+from scrapy_zyte_smartproxy.utils import exp_backoff
 
 
-class CrawleraMiddleware(object):
+class ZyteSmartProxyMiddleware(object):
 
-    url = 'http://proxy.crawlera.com:8010'
+    url = 'http://proxy.zyte.com:8011'
```

Comment:
Very very dumb question but I want to make sure - Has this been tested to make sure this works? cc @zloidemon @Gallaecio

Reply:
I had not tested it myself, no.

I’ve just tried setting `CRAWLERA_URL` to the new URL in an existing project, though, and it worked.

I think the only problems arise from switching the URL to HTTPS, as that may require installing the Zyte Smart Proxy Manager CA certificate, which is why we did not do that.